### PR TITLE
Updated UI for Edit View

### PR DESCRIPTION
**Pull request by nkompella**
Used bootstrap to create a better form, completed other UI updates. One thing to note is that these UI changes only impact the edit form as of now - we have stories this iteration that add elements to this page that will require an UI update later.

### CODE REVIEW

File: features/signup.feature [hunk 1]
```diff
@@ -14,7 +14,7 @@ Scenario: Sign up with proper credentials
 	And I fill in "user[password]" with "hello1"
 	And I fill in "user[password_confirmation]" with "hello1"
 	And I press "Sign up"
-	Then I should see "Enter dietary restrictions and preferences here"
+	Then I should see "Tell us about yourself."
```
Comment (omidafshar) [inline features/signup.feature:17]: This is definitely more user-friendly 👍

File: app/views/users/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,94 @@
+<body class="edit-view">
```
Comment (omidafshar) [inline app/views/users/edit.html.erb:1]: Interesting, what motivated the switch from .haml to .erb?

Comment (nkompella) [reply app/views/users/edit.html.erb:1]: There's a lot more online support when it comes to integrating bootstrap and flexbox with .erb files!

**Review by techprojsforfun — COMMENTED** [review]
Good stuff! one question: the last commit, e415001, was that from the merge conflict resolution on the github UI for this PR, or did you manually merge it locally? just curious

**Comment by nkompella** [comment]
I merged it locally! git pull origin master from the ui-edit-view branch. I knew there was some manual deleting of files i had to do so I thought it would be cleaner that way

**Pull request merged**